From e445686215f44c95b8b777c4c7efe5318d48505e Mon Sep 17 00:00:00 2001
From: Gustavo Zacarias <gustavo@zacarias.com.ar>
Date: Thu, 21 Jan 2016 15:14:44 -0300
Subject: [PATCH] iproute2: fix building with musl
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

We need limits.h for PATH_MAX, fixes:

rt_names.c:364:13: error: ‘PATH_MAX’ undeclared (first use in this
function)

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>
---
 lib/rt_names.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/rt_names.c b/lib/rt_names.c
index f6d17c0..b665d3e 100644
--- a/lib/rt_names.c
+++ b/lib/rt_names.c
@@ -18,6 +18,7 @@
 #include <sys/time.h>
 #include <sys/socket.h>
 #include <dirent.h>
+#include <limits.h>
 
 #include <asm/types.h>
 #include <linux/rtnetlink.h>
-- 
2.4.10

